syntax = "proto3";

message Initialize {
  string ModulePath = 1;
  string ModuleName = 2;
  string EP_File = 3;
  string EP_Function = 4;
  repeated string args = 5;
  message ConfigOptions {
    int32 LogLvl = 1;
    string DataFolder = 2;
  }
  ConfigOptions config = 6;
}

message InitializationResult {
  int32 Status = 1;
  string ErrDescription = 2;
}
